.ci/download-deps.sh

```diff
@@ -3,7 +3,7 @@
 
 function get_deps() {
 
-	TSS_VERSION=${TPM2_TSS_VERSION:-2.4.0}
+	TSS_VERSION=${TPM2_TSS_VERSION:-2.4.x}
```

Comment:
2.4.x is a branch name. Don't you want a tag here? You may use 2.4.1 instead.

Reply:
The CI errors with `tss2 pcrextend` will be fixed as soon as https://github.com/tpm2-software/tpm2-tss/pull/1811 and https://github.com/tpm2-software/tpm2-tss/pull/1812 are merged. Thus, for the tests to run successfully, a version of the TSS is needed where these fixes are included. Probably 2.4.2?

Reply:
Yes, it will be in 2.4.2, which should be out soon.

Reply:
Ok. I will update the commit as soon as it is available.

Reply:
So, I updated to tag 2.4.2. Travis is fine now, but Cirrus reports a weird error. Maybe it was only temporary. Could retrigger the build?

Reply:
On Cirrus it always runs against TSS master so it doesn't really matter what TSS version is set in .ci/download-deps.sh
but it should work just fine, as it also runs fine on Travis against TSS master: https://travis-ci.org/github/tpm2-software/tpm2-tools/jobs/715117614
We have recently checked in a change to tcti-mssim to change it to async mode. Maybe this causes some trouble in Cirrus. I will look into it.

Reply:
Anything new?